bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttZ01j_5f/ttZ01j_5f_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+import model sm-ckm_no_b_mass
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+generate p p > t t~ z, z > j j / h @0
+add process p p > t t~ z, z > vl vl~ / h @1
+add process p p > t t~ ell+ ell- / h @2
+add process p p > t t~ z j, z > j j /h @3
+add process p p > t t~ z j, z > vl vl~ / h @4
+add process p p > t t~ ell+ ell- j / h @5
+
+output ttZ01j_5f -nojpeg
```

Comment:
We are producing similar sample as it was in Run2.

Reply:
same comment here, this process may have another issue that the size of gridpack and number of file inside the gridpack is large, in Run3 we have limits on these parameters to avoid causing computing issue during production. So it's better to separate them

Reply:
Same here for the process also. I don't think that I need to follow about these change because in the last comment according to @sihyunjeon these process are being already initiated to produce. So, I removed to avoid double card being produced.